### PR TITLE
Initialize django project

### PR DESCRIPTION
**Pull request by 22shubh22**
This PR contains commit related to
* swagger auto-generate doc
* jet-dashboard
* cors

### CODE REVIEW

File: judge/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.env
```
Comment (ExpressHermes) [inline judge/.gitignore:1]: Update this file to ignore db.sqlite3 and *.pyc files. These files are generated every time the app runs.

**Review by ExpressHermes — APPROVED** [review]

File: judge/.gitignore (hunk 1 above)
Comment (ExpressHermes) [reply judge/.gitignore:1]: Update this and I will merge the changes.

Comment (22shubh22) [reply judge/.gitignore:1]: done @ExpressHermes

**Comment by 22shubh22** [comment]
@ExpressHermes recent changes contain changes related to
* question model create
* test model create
* use PostgreSQL as the database instead of SQL
* use of python-dotenv library
* Some python dependency version change

Please consider this too to review

**Review by ExpressHermes — APPROVED** [review]

**Pull request merged**